src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/editmode/PreviewEditModePageObject.java

```diff
@@ -61,17 +60,16 @@ public void verifyVideoAlignment(PositionsVideo positions) {
     );
   }
 
-  public void verifyVideoWidth(int desiredWidth) {
-    int width = Integer.parseInt(previewModal.findElement(
-        videoWidthSelector
-    ).getAttribute("width"));
-    Assertion
-        .assertNumber(width, desiredWidth, "width should be " + desiredWidth + " but is " + width);
+  public int getVideoWidth() {
+    int width = Integer.parseInt(previewModal.findElement(videoWidthSelector).getAttribute("width"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "width". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1488)